service/sandbox-core/src/test/java/de/adorsys/psd2/sandbox/xs2a/WebConfiguration.java

```diff
@@ -0,0 +1,58 @@
+package de.adorsys.psd2.sandbox.xs2a;
+
+import de.adorsys.psd2.sandbox.certificate.EnableCertificateGeneratorService;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;
+import springfox.documentation.swagger.web.*;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+import java.util.ArrayList;
+
+@EnableCertificateGeneratorService
+@EnableSwagger2
+@Configuration
+public class WebConfiguration extends WebMvcConfigurerAdapter {
```

Comment:
why do we need two configs with the same swagger?

Reply:
will it work for customer devportals and sandboxes (they are all customized at customers by external resources)?